multiqc/modules/checkqc/checkqc.py

```diff
@@ -472,6 +480,30 @@ def add_undetermined_percentage_section(self):
             plot=bargraph.plot(data, cats, pconfig),
         )
 
+    def add_undetermined_percentage_zero_yield_section(self):
+        """Add a section for lanes with zero yield
+
+        Creates a table with the lane names with zero yield
+
+        Args:
+            run (str): name of sequencing run
+        """
+        data = self.checkqc_data["UndeterminedPercentageHandler_ZeroYield"]
+
+        pconfig = {"id": "checkqc_zero-yield-table", "table_title": "CheckQC: Lanes with Yield 0", "scale": "Reds"}
+
+        headers = OrderedDict()
+        headers["lane"] = {"title": "Lane", "description": "Sequencing lane", "format": "{:,.0f}"}
+        if self.onerun:
+            headers["run"] = {"title": "Run", "description": "Sequencing run"}
```

Comment:
Shouldn't this be set if `not self.onerun`?

Reply:
Fixed in matrulda@9051fa6